### PR TITLE
Split Resque to their own containers

### LINKED ISSUE #3580: Docker-compose Works With FFmpeg Adapter in Latest Active_Encode
### Description
Updating our Docker setup to work with the local FFmpeg adapter in active_encode updates to Avalon.

### Done Looks Like
- A new worker container is made which runs the FFmpeg adapter
- The worker listens to the background job queue
- Multiple worker containers are possible 
- Writes to the Avalon/active_encode database 
- Deploys derivatives to streaming server store
- Streaming should work

### Notes
- This issue depends on code changes to avalon (#3578)
- General goal for splitting resources out for transcoding is two-fold: Docker setup which allows you to use multiple containers, and splitting FFmpeg across a separate machine or machines

### PR DESCRIPTION
**Pull request by phuongdh**
Fixes #3580 partially

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -90,53 +90,36 @@ services:
...
-      - EMAIL_NOTIFICATION
-      - EMAIL_SUPPORT
-      - ENCODE_WORK_DIR=/streamfiles
-      - FEDORA_BASE_PATH
       - FEDORA_NAMESPACE=avalon
       - FEDORA_URL=http://fedoraAdmin:fedoraAdmin@fedora:8080/fedora/rest
       - SETTINGS__FFMPEG__PATH=/usr/bin/ffmpeg
-      - MASTER_FILE_PATH
-      - MASTER_FILE_STRATEGY=delete
+      - MASTER_FILE_STRATEGY=none
```
Comment (cjcolvar) [inline docker-compose.yml]: Does it make sense to default this to `none` or should it get set back to `delete`?

Comment (phuongdh) [reply docker-compose.yml]: In development I found it easier to debug if masterfiles aren't deleted. These are visible outside of the container and can be easily managed if necessary.

Comment (cjcolvar) [reply docker-compose.yml]: But this docker-compose.yml is being used for more than just development, right?  If it is then I think we should try to mirror best practices and not what might be best for specific development tasks.  Is this a change that you can just make locally for now?

Comment (phuongdh) [reply docker-compose.yml]: This docker-compose is just used for development. Actually I'll just get rid of it and use the default settings.yml.

**Pull request merged**